recipe/build.sh

```diff
@@ -20,6 +20,46 @@ fi
 cp ${MARIADB_VERSION}/COPYING .
 mv tmp ${MARIADB_VERSION}/storage/mytile
 cd ${MARIADB_VERSION}
+
+# Save current values of the variables
+OLD_CC=$CC
```

Comment:
I don't think we need to save/restore because we are doing the tool build in a single call below.

Reply:
Yeah that's what I thought after our discussion as well but it wasn't the case. The second build was targeting x86

Reply:
Remove `export` at the beginning of the line.

Reply:
What's the status of this? I agree with Isaiah that we shouldn't need to export the variables if they are defined on the same line as the command. But I see in the commit history that you tried this and it failed. Any idea why?

Reply:
Yes indeed, I tried removing the export and it failed the CI. Essentially what happened was that it built the tools initially for x86 and then tried to build tiledb-mariadb for x86 as well

Reply:
Ok, I don't fully understand this behavior. I'll let @ihnorton decide if this strategy is acceptable